src/apis/onsite/general/guest/@types.ts

```diff
@@ -1,20 +1,8 @@
-const colors = [
-  "blue",
-  "red",
-  "yellow",
-  "violet",
-  "orange",
-  "green",
-  "gray",
-  "white",
-  "test_blue",
-  "test_red",
-  "test_yellow",
-] as const;
+import { ColorId } from "../term/@types";
 
 export interface Guest {
   id: string;
-  color_id: typeof colors[number];
+  color_id: ColorId;
```

Comment:
ここまで行ったらprefixIDに変えたほうが良くない?
```suggestion
  prefix: prefixId;
```

Reply:
てかここまで来たら color_id 要らなくないか?

Reply:
それ guest.id と term.prefix の結びつきが無くなりませんか？

TS 4.1 の Template Literal Types でできないか考えてみたけど、

```ts
import { GuestIdPrefix } from "../term/@types";

export type GuestId = `${GuestIdPrefix}-${string}`;
//                                        Template literal type argument 'string' is not literal type or a generic type.

export interface Guest {
  id: GuestId;
```

って言われて終わり

どちらにせよそこまで型に頼るのもどうなんですか？って感じがある